packages/js/product-editor/src/components/header/hooks/use-publish/use-publish.tsx

```diff
@@ -133,6 +148,7 @@ export function usePublish( {
 			: __( 'Add', 'woocommerce' ),
 		...props,
 		isBusy,
+		disabled: isDisabled,
```

Comment:
Hey, I feel like we shouldn't use 'disabled' directly. I suggested that we should have the same behavior as the post editor, it uses `aria-disabled` to achieve the same behavior. That way the button is accessible to screen readers.

Reply:
Nice. Addressed here https://github.com/woocommerce/woocommerce/pull/41473/commits/a75fa7a6afd7e707234fa656aa51bcf76a75fb39

I caught the state in the following screenshot:


<img width="865" alt="Screenshot 2023-11-17 at 16 28 35" src="https://github.com/woocommerce/woocommerce/assets/77539/984d131e-9684-479d-8e6a-ccc5b6473ff2">